spec/docker/container_spec.rb

```diff
@@ -490,6 +490,42 @@
         id.start_with?(subject.id)
       }
     end
+
+    context 'with a timeout' do
+      let(:custom_timeout) { 60 }
+
+      before do
+        subject.tap(&:start)
+      end
+
+      it 'extends the Excon HTTP timeout ensuring the HTTP request does not timeout before Docker' do
+        expect(subject.connection).to receive(:request).with(
+          :post,
+          anything,
+          anything,
+          hash_including(read_timeout: custom_timeout + 5, write_timeout: custom_timeout + 5)
+        ).once
+        allow(subject.connection).to receive(:request).with(:delete, any_args)
```

Comment:
Running locally, it looks like `any_args` needs to be changed to `anything, anything`.